recipes/epics-pypdb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "epics-pypdb" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/epics-pypdb/archive/{{ version }}.tar.gz
+  sha256: 3e402b8f946c528257203649dc1e33ac49d56773d6b1ff7df40407a478cf9900
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```

Comment:
you don't need pip and setuptools. Please remove setuptools

Reply:
Thank you, @scopatz for the review.
This was addressed with the latest commit.